rskj-core/src/main/java/co/rsk/peg/ErpRedeemScriptBuilderUtils.java

```diff
@@ -0,0 +1,35 @@
+package co.rsk.peg;
+
+import co.rsk.bitcoinj.core.NetworkParameters;
+import org.ethereum.config.blockchain.upgrades.ActivationConfig;
+import org.ethereum.config.blockchain.upgrades.ConsensusRule;
+
+public class ErpRedeemScriptBuilderUtils {
```

Comment:
This is actually like a NonStandardErpRedeemScript factory actually, right?

Reply:
yes, absolutely